pettingzoo/utils/env.py

```diff
@@ -262,17 +262,17 @@ def __iter__(self):
 
 
 class AECIterator(Iterator):
-    def __init__(self, env, max_iter):
+    def __init__(self, env: AECEnv, max_iter: int) -> None:
         self.env = env
         self.iters_til_term = max_iter
 
-    def __next__(self):
+    def __next__(self) -> str:
```

Comment:
I’m not 100% sure agent selection has to be a string cause the user can input it as whatever they want

Reply:
Also like with init I’ve never seen return type specified for double underscore methods like this